TombEngine/Sound/sound.cpp

```diff
@@ -90,10 +102,12 @@ bool LoadSample(char* pointer, int compSize, int uncompSize, int index)
 	BASS_SampleGetInfo(sample, &info);
 	int finalLength = info.length + 44;	// uncompSize is invalid after 16->32 bit conversion
 
-	if (info.freq != 22050 || info.chans != 1)
+
 	{
-		TENLog("Wrong sample parameters, must be 22050 Hz Mono", LogLevel::Error);
-		return false;
```

Comment:
why was `return false` deleted?

the return false makes sure it will exit the function if it couldn't load samples with either frequency (I haven't looked at the function but by the if statement and function name I can guess :p).

Reply:
done.